### PR TITLE
improve delete by labeling created docker volumes 

### PR DESCRIPTION
**Pull request by medyagh**
closes https://github.com/kubernetes/minikube/issues/6662
might also close https://github.com/kubernetes/minikube/issues/6646

### CODE REVIEW

File: cmd/minikube/cmd/delete.go [hunk 1]
```diff
@@ -97,7 +98,7 @@ func runDelete(cmd *cobra.Command, args []string) {
 
 	validProfiles, invalidProfiles, err := pkg_config.ListProfiles()
 	profilesToDelete := append(validProfiles, invalidProfiles...)
-
+	glog.Infof("error listing profiless %v", err)
```
Comment (afbjorklund) [inline cmd/minikube/cmd/delete.go]: typo, and should probably check for nil ?

File: cmd/minikube/cmd/delete.go [hunk 2]
```diff
@@ -112,8 +113,9 @@ func runDelete(cmd *cobra.Command, args []string) {
 			exit.UsageT("usage: minikube delete --all")
 		}
 
-		if err != nil {
```
Comment (afbjorklund) [inline cmd/minikube/cmd/delete.go]: still need to check for the old err, before checking for the new ?

File: cmd/minikube/cmd/delete.go (hunk 1 above)
Comment (medyagh) [reply cmd/minikube/cmd/delete.go]: thanks done

**Pull request merged**